ws/src/speech/scripts/UsefulAudio.py

```diff
@@ -38,33 +38,36 @@
 
 current_dir = os.path.dirname(os.path.abspath(__file__))
 
+
 class Timer():
     def __init__(self):
         self.timer = rospy.Time.now()
 
     def startTime(self):
         self.timer = rospy.Time.now()
-    
+
     def endTimer(self):
         time_delta = rospy.Time.now() - self.timer
         time_delta_second = time_delta.to_sec()
         return time_delta_second
 
+
 class UsefulAudio(object):
     triggered = False
     chunk_count = 0
     voiced_frames = []
-    ring_buffer = collections.deque(maxlen = NUM_PADDING_CHUNKS)
+    ring_buffer = collections.deque(maxlen=NUM_PADDING_CHUNKS)
 
-    audioCollection = [] # Temporary storage for audio data, to be used in the VAD
-    chunkCollection = None # Temporary storage for chunk data, to be sent to the ASR model
+    audioCollection = []  # Temporary storage for audio data, to be used in the VAD
+    chunkCollection = None  # Temporary storage for chunk data, to be sent to the ASR model
 
     def __init__(self, threshold: float = 0.1):
         if not USE_SILERO_VAD:
             self.vad = webrtcvad.Vad()
             self.vad.set_mode(3)
         else:
-            model_path = os.path.join(current_dir, "../assets", "silero_vad.onnx")
+            model_path = os.path.join(
+                current_dir, "../assets", "silero_vad.onnx")
             options = onnxruntime.SessionOptions()
             options.log_severity_level = 4
 
```

Comment:
```suggestion
            model_path = os.path.join(
                current_dir, "../assets", "silero_vad.onnx")
            options = onnxruntime.SessionOptions()
            options.log_severity_level = 4
            if not os.path.exists(model_path):
                throw Exception("Model not found")
```

Reply:
Thanks for the suggestion, but will leave as is because the library's message is sufficiently self-explanatory:

`process[UsefulAudio-6]: started with pid [4042]
Traceback (most recent call last):
  File "/workspace/ws/src/speech/scripts/UsefulAudio.py", line 330, in <module>
    main()
  File "/workspace/ws/src/speech/scripts/UsefulAudio.py", line 323, in main
    usefulAudio = UsefulAudio(threshold=THRESHOLD)
  File "/workspace/ws/src/speech/scripts/UsefulAudio.py", line 77, in __init__
    self.inference_session = onnxruntime.InferenceSession(
  File "/usr/local/lib/python3.8/dist-packages/onnxruntime/capi/onnxruntime_inference_collection.py", line 419, in __init__
    self._create_inference_session(providers, provider_options, disabled_optimizers)
  File "/usr/local/lib/python3.8/dist-packages/onnxruntime/capi/onnxruntime_inference_collection.py", line 452, in _create_inference_session
    sess = C.InferenceSession(session_options, self._model_path, True, self._read_config_from_model)
onnxruntime.capi.onnxruntime_pybind11_state.NoSuchFile: [ONNXRuntimeError] : 3 : NO_SUCHFILE : Load model from /workspace/ws/src/speech/scripts/../assets/silero_vadt.onnx failed:Load model /workspace/ws/src/speech/scripts/../assets/silero_vad.onnx failed. File doesn't exist`